java-desktop/src/main/java/com/couchbase/todo/controller/MainController.java

```diff
@@ -12,13 +12,20 @@
 import com.couchbase.todo.TodoApp;
 import com.couchbase.todo.model.DB;
 
+
 public class MainController implements Initializable {
 
-    @FXML private TaskListsController taskListsController;
+    @FXML
```

Comment:
Please be consistent with the existing code.
I, actually, *prefer* putting the annotation on a separate line.  If you are gonna do that, though,
please go fix the other files too.  I would like consistent formatting within a project.

Reply:
Thanks for fixing this.  I suspect I wrote this code, so it is me you are cleaning up after!

Much appreciated.